### PR TITLE
Update to support validating 1.19.2 minecraft-data protocol.json

### PR DESCRIPTION
**Pull request by jtsiskin**

### CODE REVIEW

File: lib/plugins/chat.js [hunk 1]
```diff
@@ -131,10 +131,19 @@ function inject (bot, options) {
       const { profileKeys } = bot.players[sender]
       if (profileKeys) verified = bot._client.verifyMessage(profileKeys.publicKey, packet)
     }
-    const parameters = {
-      sender: JSON.parse(packet.senderName),
-      content: JSON.parse(message)
+    let parameters = {};
```
Comment (Kashalls) [inline lib/plugins/chat.js:134]: This code is messy.

Consider doing something like:
```js
const parameters = {
    sender: JSON.parse(packet.networkName ? packet.networkName: packet.senderName),
    content: JSON.parse(packet.networkName ? { "text": packet.plainMessage } : message)
  }

Comment (rom1504) [reply lib/plugins/chat.js:134]: I think the code in the PR is better actually. Makes more sense to branch on this once.
Could maybe be improved by using support feature though

Comment (jtsiskin) [reply lib/plugins/chat.js:134]: This commit in general is just "here's the minimum diff to let us verify the minecraft-data changes and make it easier to continue development"

**Review by liuk3 — APPROVED** [review]

File: lib/plugins/chat.js (hunk 1 above)
Comment (Shoro123456789) [reply lib/plugins/chat.js:134]: how long would it take  for this bot to update to 1.19.2

**Comment by rtm516** [comment]
Fixed /tell in https://github.com/jtsiskin/mineflayer/pull/1

**Comment by rom1504** [comment]
this is not working

**Comment by rom1504** [comment]
another pr doing it

**Pull request closed without merging**